ios/FluentUI.Demo/FluentUI.Demo/Demos/ObjectiveCDemoController.m

```diff
@@ -63,6 +63,30 @@ - (void)viewDidLoad {
 
     MSFButton *resetButton = [self createButtonWithTitle:@"Reset Vnext Button" action:@selector(resetVnextButton)];
     [self.container addArrangedSubview:resetButton];
+    
+    UILabel *listVnextLabel = [[UILabel alloc] init];
+    [listVnextLabel setText:@"List (vNext)"];
+    [self.container addArrangedSubview:listVnextLabel];
+    
+    MSFListVnextCellData *listCell1 = [[MSFListVnextCellData alloc] init];
+    listCell1.title = @"Sample Title1";
+    MSFListVnextCellData *listCell2 = [[MSFListVnextCellData alloc] init];
+    listCell2.title = @"Sample Title2";
+    listCell2.subtitle = @"Sample Subtitle2";
+    MSFListVnextCellData *listCell3 = [[MSFListVnextCellData alloc] init];
+    listCell3.title = @"Sample Title3";
+    listCell3.subtitle = @"Sample Subtitle3";
+    UIImage *image = [UIImage imageNamed:@"excelIcon"];
+    listCell3.leadingIcon = image;
+    listCell3.trailingIcon = MSFListAccessoryTypeDisclosure;
+    
+    MSFListVnextSectionData *section = [[MSFListVnextSectionData alloc] init];
+    section.layoutType = MSFListCellVnextHeightTwoLines;
+    section.cells = @[listCell1, listCell2, listCell3];
+    NSArray *sections = @[section];
+
+    MSFListVnext *list = [[MSFListVnext alloc] initWithSections:sections iconStyle:MSFListIconVnextStyleNone];
+    [self.container addArrangedSubview:[list view]];
```

Comment:
Can you fix the constraints here in the same way they were done on the List Demo Controller?